peachjam/views/courts.py

```diff
@@ -49,10 +49,11 @@ def get_context_data(self, **kwargs):
             context["grouped_documents"] = self.grouped_judgments(context["documents"])
 
         if "year" in self.kwargs:
-
             context["year"] = self.kwargs["year"]
 
-            if "month" in self.kwargs:
+            if "month" in self.kwargs and int(self.kwargs["month"]) in set(
```

Comment:
Shouldn't self.kwargs['month'] already be an int? Otherwise `MONTHS[self.kwargs["month"]]` wouldn't work.